frontend-service/api/swagger.yml

```diff
@@ -428,4 +428,4 @@ definitions:
         type: "string"
       newTargetRevisionId:
         type: "string"
-x-generated-at: "2018-01-09T22:55:42.535Z"
+x-generated-at: "2018-01-09T22:55:42.535Z"
```

Comment:
This is auto-updated by [this filter](https://github.com/blox/blox/blob/dev/buildSrc/src/main/java/com/amazonaws/blox/swagger/GenerationTimestampFilter.java), which seemed like a good idea at the time, but now seems like it just makes annoying noise in our diffs.

I think we can remove that filter in a later PR to avoid this issue in future.